plugins/verifier/schemavalidator/schemavalidation/schemavalidator.go

```diff
@@ -42,3 +42,22 @@ func Validate(schema string, content []byte) error {
 		return errors.New(e)
 	}
 }
+
+func Validate2(schema []byte, content []byte) error {
```

Comment:
When should we use Validate2()  compared to Validate(), Suggestion to add comments to Validate and Validate2 public methods.